tests/fixtures/controllers/methodController.ts

```diff
@@ -145,6 +145,7 @@ export class MethodController extends Controller {
   }
 
   @Get('returnAliasedVoidType')
+  // tslint:disable-next-line:no-empty
   public async returnAliasedVoidType(): Promise<VoidAlias1> {}
```

Comment:
I assume this should avoid any issues with tslint:
```suggestion
  public async returnAliasedVoidType(): Promise<VoidAlias1> {
    return;
  }
```

Reply:
It is not my mistake, but i think to disable rule is more clean fix, then adding some redundant code that only aims to disable rule and does nothing, but i can fix like you suggest.

Reply:
I disagree, we should conform to the coding style tslint suggests unless we have a very good reason not to.
I know you didn't introduce the issue, but if you want to address it, this would be the better way.

Reply:
Ok, fixed.